### PR TITLE
Add both tf.nn.X and tf.compat.v1.nn.X to activation_map

### PR DESCRIPTION
**Pull request by jiafatom**
The [issue](https://github.com/onnx/keras-onnx/issues/511) cannot convert softmax activation because `operator.raw_operator.activation` is a function that is not the same as `tf.nn.softmax` or `keras.activations.get('softmax')`. We need import `tf` from `keras2onnx.proto.tfcompat`

### CODE REVIEW

File: keras2onnx/ke2onnx/activation.py [hunk 1]
```diff
@@ -34,7 +34,16 @@
                   tf.nn.relu: apply_relu,
                   tf.nn.relu6: apply_relu_6,
                   tf.nn.elu: apply_elu,
-                  tf.nn.tanh: apply_tanh}
+                  tf.nn.selu: apply_selu,
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/activation.py:37]: any idea why the function is not as tf.nn? if it was customized function was created by the user, it will be pretty dangerous to  use the name.

Comment (jiafatom) [reply keras2onnx/ke2onnx/activation.py:37]: Through debugging, I can only see the function has different address from tf.nn.softmax.

Comment (jiafatom) [reply keras2onnx/ke2onnx/activation.py:37]: re-checked, code updated.

File: keras2onnx/ke2onnx/activation.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # license information.
 ###############################################################################
 import numpy as np
-import tensorflow as tf
+from keras2onnx.proto.tfcompat import tensorflow as tf
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/activation.py:7]: good finding, you could be add both tensorflow.compat.v1.nn and tensorflow.nn

Comment (jiafatom) [reply keras2onnx/ke2onnx/activation.py]: good, updated.

File: keras2onnx/ke2onnx/activation.py [hunk 3]
```diff
@@ -34,7 +34,15 @@
                   tf.nn.relu: apply_relu,
                   tf.nn.relu6: apply_relu_6,
                   tf.nn.elu: apply_elu,
-                  tf.nn.tanh: apply_tanh}
+                  tf.nn.selu: apply_selu,
+                  tf.nn.tanh: apply_tanh,
+                  tf.compat.v1.nn.sigmoid: apply_sigmoid,
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/activation.py:39]: need check tf version before add it.

Comment (jiafatom) [reply keras2onnx/ke2onnx/activation.py]: checking hasattr(tf.compact, 'v1')
